src/openslide-vendor-dicom.c

```diff
@@ -398,7 +421,7 @@ static void rgb_to_cairo(const uint8_t *rgb, uint32_t *dest,
                          int64_t width, int64_t height) {
   int64_t n_pixels = width * height;
   for (int64_t i = 0; i < n_pixels; i++) {
-    dest[i] = 0xff000000 | rgb[0] << 16 | rgb[1] << 8 | rgb[2];
+    dest[i] = 0xff000000 | rgb[2] << 16 | rgb[0] << 8 | rgb[1];
```

Comment:
I agree that this makes the images render correctly, but GBR is a weird channel order, and is not how `PlanarConfiguration` 0 is documented.  Can we try to track down the underlying cause?  Is it possible that there's an off-by-one error somewhere?

Reply:
I agree it does look odd. 

I'm reworking pixeldata parsing right now, so let's circle back to this once that is merged.

Reply:
OK, I think the new pixeldata code has fixed this.